UnitsNet/Scripts/UnitDefinitions/Ratio.json

```diff
@@ -15,6 +15,18 @@
                 }
             ]
         },
+	{
+            "SingularName": "ComplementDecimalFraction",
+            "PluralName": "ComplementDecimalFractions",
+            "FromUnitToBaseFunc": "1-x",
+            "FromBaseToUnitFunc": "1-x",
+            "Localization": [
+                {
+                    "Culture": "en-US",
+                    "Abbreviations": [""]
```

Comment:
Same here, as below. Duplicate abbreviation. Need a strategy here and add some tests to verify the right unit is actually parsed if we do need to share abbreviations.